### DISCUSSION TITLE
Given Redis's license change, do we have any plan to let NestJs extend bullMQ to support Valkey?

### DISCUSSION THREAD

**Discussion by shenmufeng** [taskforcesh/bullmq#2642] [Ideas]
According to Redis's license change, I think supporting Valkey becomes a must.
Do we have any plan to update bullMQ to support Valkey in the future?
When should we expect to see it?
Thank you so much~!

**Comment by manast** [comment]
I guess it is the other way around, if Valkey is 100% compatible with Redis then it will work with BullMQ, if not, then maybe not. There are many Redis alternatives these days, if they really are drop-in replacements they would work.

**Comment by Bjoernstjerne** [comment]
@manast do you have more information about your twitter post?
https://x.com/manast/status/1841042438510494090
Found it here:
https://github.com/taskforcesh/bullmq/pull/2794
Thank you for testing.

**Reply by roggervalf** [reply]
hi @Bjoernstjerne that pr means that Valkey is compatible with bullmq as all our tests are passing
